docs/content/docs/start/tree-start.md

```diff
@@ -0,0 +1,239 @@
+
+This is how we would initialize our todo list:
+
+```
+appData.initialize(new TodoList({
+	title: "todo list",
+	items: [
+		new TodoItem({
+			description: "first item",
+			isComplete: true,
+		}),
+	]
+}));
+```
+
+## Reading Data From Your Tree
+
+Data can be read from the tree using the `root` property of the `TreeView` that you obtained through the `viewWith` method.
+Since the root of our tree is an object schema, its data can be accessed like a regular JavaScript object.
+
+If `TodoList` is the root schema of your tree, you can access the todo list data like this:
+
+```
+const todoListTitle = appData.root.title;
+const firstItem = appData.root.items[0];
+```
+
+You can also iterate through the actual items, here's an example of how to use the data you're reading in a JSX component:
+
+```
+<li>
+	{appData.root.items.map((item) =>
+		<ul>{item.description}</ul>
+	)}
+</li>
+```
+
+When using the data in your tree, it's important to listen to the change events provided by the tree so that you can make any necessary updates to your application.
+This is how you can use React hooks to listen to the `nodeChanged` event:
+
+```
+const [itemCount, setCount] = useState(appData.root.items.length);
+
+useEffect(() => {
+	const unsubscribe = Tree.on(appData.root.items, "nodeChanged", () => {
+		setCount(appData.root.items.length);
+	});
+	return unsubscribe;
+}, []);
+```
+
+`nodeChanged` fires whenever one or more properties of the specified node change while `treeChanged` also fires whenever any node in its subtree changes.
+See [the API](../api/v2/tree/treechangeevents-interface) docs for more details.
+
+## Editing Tree Data
+
+There are built-in editing methods for each of the provided schema types.
+For example, if your data is in an array, you can add an item like this:
+
+```
+appData.root.items.insertAt(3);
+```
+
+The schema types can also be edited using the assignment operator like this:
+
+```
+appData.root.title = "chores";
+```
+
+Editing methods can also be defined on schema classes defined in customizable mode.
+This allows you to write methods that are more suited to your app's specific needs.
+
+```
+class TodoList extends schemaFactory.object("TodoList", {
+	title: schemaFactory.string,
+	items: todoItems,
+}) {
+	public removeFirst = () => {
+		if (this.length > 0) this.removeAt(0);
+	};
+}
+```
+\
+See [schema definition](../data-structures/tree#schema-definition) for more details on the built-in editing methods.
+You can also read more about how these editing operations work in collaborative settings [here](https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/docs/main/merge-semantics.md).
+
+### Grouping Edits into Transactions
+
+Edits can be grouped into transaction using the `Tree.runTransaction()`	API.
+Using a transaction guarantees that (if applied) all of the changes will be applied together synchronously.
```

Comment:
So, interestingly enough fluid already guarantees that all edits made in the same JavaScript turn will be applied synchronously so this is not a guarantee worth mentioning. The primary benefit to customers right now is the fact that transactions are the unit of undo redo. When we eventually expose branching, this will also have implications for how many commits show up in a branch (as transactions are essentially squashing) but for now, outside of undo/redo, the only other benefit is some poorly defined performance improvements of squashing that probably shouldn't be mentioned in this doc.